examples/python/helloworld/helloworld_pb2_grpc.py

```diff
@@ -1,9 +1,38 @@
 # Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
 """Client and server classes corresponding to protobuf-defined services."""
 import grpc
+import warnings
 
 import helloworld_pb2 as helloworld__pb2
 
+GRPC_GENERATE_VERSION='1.62.0.dev0'
+GRPC_VERSION=grpc.__version__
+
+def _first_version_is_lower(version1, version2):
```

Comment:
Can we make this a protected API within `grpc` itself and just _attempt_ to load it here? If not, this seems like it's going to result in a _lot_ of duplication.

If the import fails, we can assume that the version of grpc is _older_ than `grpcio-tools` and throw the warning.